scripts/Diffraction/single_crystal/test/test_sxd.py

```diff
@@ -20,7 +20,9 @@
 from Diffraction.single_crystal.sxd import SXD
 from Diffraction.single_crystal.base_sx import PEAK_TYPE, INTEGRATION_TYPE
 
-sxd_path = "Diffraction.single_crystal.sxd"
+diffraction_path = "Diffraction.single_crystal"
+sxd_path = diffraction_path + ".sxd"
+base_sx_path = diffraction_path + ".base_sx"
```

Comment:
Not a convention I was aware of sorry! I have seen upper case constants as attributes of e.g. algorithm classes - but didn't realise this applied elsewhere. I will set to upper case.

Reply:
Yes, I tend to do it but wasn't totally sure if it was a convention so I didn't some digging in the docs and found [this](https://developer.mantidproject.org/Standards/PythonStandards.html#style) which says we're using PEP 8 standards and those say [this](https://peps.python.org/pep-0008/#constants) about constants. So I thought it might be worth mentioning.

Reply:
Good to know, thanks!